examples/demo_cisco_tacacs_server.pp

```diff
@@ -0,0 +1,27 @@
+# Manifest to demo cisco_tacacs_server provider
```

Comment:
@mikewiebe I will rename our files. 

Please create the _all.pp structure in another PR and add our files to it.